src/condor_utils/dagman_utils.h

```diff
@@ -279,6 +351,9 @@ class DagmanOptions {
 	DagOptionData<SSO> shallow;
 	//Deep options passed down to subdags
 	DagOptionData<DSO> deep;
+	// Map of used bool options to prevent contradictary flags
+	// Only applies to AutoParse()
+	std::map<std::string, std::string> boolFlagCheck;
```

Comment:
This seems like it could be, and would be better off being, a local in `AutoParse()`.